### ISSUE TITLE
VBoxManage.exe: error: Failed to open/create the internal network 'HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter

### ISSUE THREAD

**Issue by ahusmc** [rapid7/metasploitable3#429]
## Issue Description
When running `vagrant up win2k8` I'm getting the following error:

There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "b22adc26-28a4-41a1-96c7-0a6ea64d6094", "--type", "headless"]

Stderr: VBoxManage.exe: error: Failed to open/create the internal network 'HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter' (VERR_INTNET_FLT_IF_NOT_FOUND).
VBoxManage.exe: error: Failed to attach the network LUN (VERR_INTNET_FLT_IF_NOT_FOUND)
VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole

## Host System
* OS: Windows 10 Pro 10.0.18363
* Packer Version: 1.4.5
* Vagrant Version: 2.2.6
* VirtualBox Version: 6.0.14 r133895 (Qt5.6.2)

## Command Output
Copy the relevant command output here.
If it's long, either post to a gist and add the link here, or isolate the error lines.

**Comment by ahusmc** [comment]
I changed the host-only NIC to bridged and then ran vagrant up. This allowed the VM to start and the config script to successfully run and complete.

**Comment by deargle** [comment]
In my virtualbox experience, the onus is on the virtualbox user to first
configure a host-only adapter, from within the virtualbox manager
interface, independent of attempting to use it. IIRC virtualbox auto
installs one at initial install time, or at least once upon a time it did.

**Comment by ahusmc** [comment]
Hi, @deargle 

Thanks for the input. Yeah, it looks like the installer config attempts to create the host-only adapter, does so, but then failed to successfully configure it. I had a number of host-only adapters created in my VirtualBox Host Network Manager and only one deployed and in use from a previous VM. Looks like a new HO adapter was created every time I tried the Metsploitable 3 deploy.

However, I was able to work through the deployment issue by simply changing the HO adapter in use by the MS3 VM to bridged mode and then booting the VM. So far so good.
